demo/Snakefile

```diff
@@ -38,14 +40,16 @@ configfile: "config.yaml"
 
 # Use all samples as a default sample list for each module
 config["lcr-modules"]["_shared"]["samples"] = SAMPLES
-
+config["lcr-modules"]["picard_qc"]["samples"] = SAMPLES
```

Comment:
Why is this necessary? Shouldn't the module inherit from `_shared` if `samples` in `picard_qc` is left unset?

Reply:
I think It's because I need `CFG["samples"]` in the `_get_sample_metrics` functions and the function is evaluated before the `op.setup_module`. So when I try to run it, I receive a KeyError that "samples" does not exist. I'm open to better ideas to go around this because I prefer not having to specify it in the project snakefile.